_data/students.yml

```diff
@@ -546,7 +546,7 @@
   twitter: Pro_MYScrub
   bio: Can you differentiate the useful and the useless?
 
-- name: Yee Jia Shuen
+- name: Yee J. S.
```

Comment:
@marionmok Why have you changed this? Please don't change other students info. Thanks!